### PR TITLE
Teams pagination

### PR DESCRIPTION
**Pull request by fede1608**
## Summary

Added pagination to `GET /teams`

### CODE REVIEW

**Review by wleoguty91 — DISMISSED** [review]

**Review by betoobernal — DISMISSED** [review]

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('dummy test', () => {
```
Comment (gonziesc) [inline __tests__/app.test.js]: whats this doing here?

**Review by gonziesc — DISMISSED** [review]

File: __tests__/app.test.js (hunk 1 above)
Comment (fede1608) [reply __tests__/app.test.js]: CI won't pass without at least one test file

**Review by wleoguty91 — APPROVED** [review]

File: .eslintrc.js [hunk 2]
```diff
@@ -1 +1,5 @@
 module.exports = require('eslint-config-wolox-node');
+
+module.exports.rules.indent = ['off']
```
Comment (gonziesc) [inline .eslintrc.js]: why this?

Comment (fede1608) [reply .eslintrc.js]: There is a conflict between this rule and prettify

**Review by gonziesc — APPROVED** [review]

**Review by gonziesc — APPROVED** [review]

**Pull request closed without merging**